.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: python
 python:
-  - "3.5"
   - "3.6"
```

Comment:
We should really test with all supported versions of Python (so up to 3.9 now).

Reply:
There are no versions of Django that support Python 3.9 yet, so I'll just include master.